assignments/prototypes.js

```diff
@@ -83,28 +150,114 @@
       height: 4,
     },
     healthPoints: 10,
-    name: 'Lilith',
-    team: 'Forest Kingdom',
+    name: 'Rowen Hashiba',
+    team: 'Ronin Warriors',
     weapons: [
       'Bow',
       'Dagger',
     ],
-    language: 'Elvish',
+    language: 'Ancient Japanese',
   });
 
   console.log(mage.createdAt); // Today's date
   console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
   console.log(swordsman.healthPoints); // 15
-  console.log(mage.name); // Bruce
-  console.log(swordsman.team); // The Round Table
-  console.log(mage.weapons); // Staff of Shamalama
-  console.log(archer.language); // Elvish
-  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
-  console.log(mage.takeDamage()); // Bruce took damage.
-  console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+  console.log(mage.name); // Kento Rei Fang
+  console.log(swordsman.team); // Ancient Warriors
+  console.log(mage.weapons); // Staff
+  console.log(archer.language); // Ancient Japanese
+  console.log(archer.greet()); // Rowen Hashiba offers a greeting in Ancient Japanese.
+  console.log(mage.takeDamage()); // Kento Rei Fang took damage.
+  console.log(swordsman.destroy()); // Sage Date was removed from the game.
```

Comment:
this line is coming back as undefined, you should go back and take a look at this one.